### PR TITLE
Feature/27 test controller

### PR DESCRIPTION
**Pull request by stishchenko**
I did this branch from Configure_SpringBoot branch.

### CODE REVIEW

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
```
Comment (rdovgan) [inline .mvn/wrapper/maven-wrapper.properties:1]: @stishchenko please add `.mvn/` folder to .gitignore file

File: mvnw [hunk 2]
```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
+# ----------------------------------------------------------------------------
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```
Comment (rdovgan) [inline mvnw:4]: This file should be also ignored

File: mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
+@REM Licensed to the Apache Software Foundation (ASF) under one
+@REM or more contributor license agreements.  See the NOTICE file
```
Comment (rdovgan) [inline mvnw.cmd:3]: Ignore this file in .gitignore

File: src/main/java/team/asd/Message.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
+package team.asd;
```
Comment (rdovgan) [inline src/main/java/team/asd/Message.java]: Move this class to `entity` package

File: src/main/java/team/asd/TestController.java [hunk 5]
```diff
@@ -0,0 +1,26 @@
+package team.asd;
```
Comment (rdovgan) [inline src/main/java/team/asd/TestController.java]: @stishchenko move this class to `controller` package

**Review by rdovgan — APPROVED** [review]

**Review by vasylnikorych — APPROVED** [review]

**Pull request merged**